test.c

```diff
@@ -239,6 +238,24 @@ test_buffer_prepend_issue_15() {
   buffer_free(file);
 }
 
+void
+test_ends_w() {
+  buffer_t *buf = buffer_new_with_copy("Good C string library btw.");
+  assert(1 == buffer_ends_w(buf, "btw."));
+  assert(1 == buffer_ends_w(buf, "C string library btw."));
+  assert(0 == buffer_ends_w(buf, "anyway."));
```

Comment:
A few more tests to improve coverage.
```suggestion
  assert(0 == buffer_ends_w(buf, "anyway."));
  assert(0 == buffer_ends_w(buf, ""));
  assert(0 == buffer_ends_w(buf, "Good C string library btw."));
  assert(0 == buffer_ends_w(buf, "This is a text that is longer than what is on the buffer."));
```